backend/src/test/java/com/woowacourse/momo/service/meeting/MeetingServiceTest.java

```diff
@@ -87,4 +89,76 @@ void findByUUID() {
         softAssertions.assertThat(result.availableDates().get(0)).isEqualTo(LocalDate.now().minusDays(1));
         softAssertions.assertAll();
     }
+
+    @DisplayName("약속 정보와 참가자 정보를 통해 약속을 등록한다.")
+    @Test
+    void create() {
+        //given
+        MeetingCreateRequest request = new MeetingCreateRequest(
+                "momoHost",
+                "momo",
+                "momoMeeting",
+                8,
+                List.of(LocalDate.of(2024, 7, 24), LocalDate.of(2024, 7, 25)),
+                LocalTime.of(8, 0),
+                LocalTime.of(22, 0));
+
+        //when
+        meetingService.create(request);
+
+        //then
+        List<Meeting> meetings = meetingRepository.findAll();
+        List<Attendee> attendees = attendeeRepository.findAll();
+        List<AvailableDate> availableDates = availableDateRepository.findAll();
+
+        assertAll(
+                () -> assertThat(meetings).hasSize(1),
+                () -> assertThat(attendees).hasSize(1),
+                () -> assertThat(availableDates).hasSize(2)
+        );
+    }
+
+    @DisplayName("약속을 생성한 참가자의 역할은 호스트이다.")
+    @Test
+    void createdMeetingByHost() {
+        //given
+        MeetingCreateRequest request = new MeetingCreateRequest(
+                "momoHost",
+                "momo",
+                "momoMeeting",
+                8,
+                List.of(LocalDate.of(2024, 7, 24), LocalDate.of(2024, 7, 25)),
+                LocalTime.of(8, 0),
+                LocalTime.of(22, 0));
+
+        //when
+        meetingService.create(request);
+
+        //then
+        List<Meeting> meetings = meetingRepository.findAll();
+        Attendee attendee = attendeeRepository.findByMeetingAndName(meetings.get(0), new AttendeeName("momoHost"))
+                .orElseThrow();
+
+        assertThat(attendee).extracting("role").isEqualTo(Role.HOST);
```

Comment:
`extracting()`메서드는 자바 리플랙션을 이용하여 동일한 이름의 필드로 직접 접근하는데요.
만약 `role`이라는 필드명이 변경될 경우 컴파일 시점에 에러를 잡아주지 못할 것 같아요!
getter로 변경하여 컴파일 시점에 오류를 잡아주는건 어떨까요?
```suggestion
        assertThat(attendee.getRole()).isEqualTo(Role.HOST);
```

Reply:
오 그건 처음 알았네요! 반영하겠습니다👍